packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1065,7 +1069,29 @@ export class MergeTree {
 			return this.getPosition(refPos, refSeq, clientId);
 		}
 		if (refTypeIncludesFlag(refPos, ReferenceType.Transient) || seg.localRefs?.has(refPos)) {
-			const offset = isRemoved(seg) || isMoved(seg) ? 0 : refPos.getOffset();
+			// for references that slide backwards, we want to take into account
+			// local/unacked changes
+			if (isRemovedOrMoved(seg) && refPos.slidingPreference === SlidingPreference.BACKWARD) {
+				let slidToSegment: ISegment | undefined;
+
+				backwardExcursion(seg, (segment) => {
```

Comment:
is this behavior actually any different from if we just compute the position of the removed segment and subtract 1 if it's >0?

Reply:
Resolving `refPos.getSegment()` to a local position already accounts for any segments removed in the current view (i.e. *doesn't* count them)